fineract-provider/build.gradle

```diff
@@ -179,13 +136,6 @@ if (project.hasProperty('env') && project.getProperty('env') == 'eclipse')
     sourceSets.main.java.outputDir = new File(rootProject.projectDir, "fineract-provider/bin/main")
 }
 
-eclipse
-{
-    project {
-        buildCommand([ LaunchConfigHandle: "<project>/.externalToolBuilders/OpenJPA Enhance Builder.launch" ],  'org.eclipse.ui.externaltools.ExternalToolBuilder')
```

Comment:
so this gets entirely ditch, not just moved.. why? @ptuomola does this LGTY? (I have NOT pulled it myself yet, to actually test.)

Reply:
... because "fineract-provider/.externalToolBuilders/OpenJPA Enhance Builder.launch" gets picked up automatically by Eclipse when you import via "Existing Gradle Project"... when I open the .project file in fineract-provider then I see this:

![](https://user-images.githubusercontent.com/380079/95926988-e791bc80-0dbd-11eb-8ae8-37bcb0840de8.png)

So... should be fine.